ui/src/app/groupings/[selectedGrouping]/_components/grouping-header.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import DescriptionForm from '@/app/groupings/[selectedGrouping]/_components/description-form';
+
+const GroupingHeader = ({ groupName, groupDescription, groupPath }: { groupName: string, groupDescription: string, groupPath: string }) => {
+    return (
+        <div className="py-3 px-5 border-b box-border rounded-b-none rounded-t mt-4 mt-0 bg-uh-teal">
+            <div className="flex flex-row table m-auto w-full p-0">
+                <div className="md:w-2/3 table-footer-group">
+                    {groupName && (
+                        <h2 className="text-gray-100 mb-0 text-[2rem] text-center md:text-left">
+                            {groupName}
+                        </h2>
+                    )}
```

Comment:
There should be no way a groupName is undefined, so the `groupingName &&` can be removed.